### DISCUSSION TITLE
Why can't we use 'for' in GridLayout？

### DISCUSSION THREAD

**Discussion by Jaydar** [slint-ui/slint#4299] [General]
If I can't use 'for', how should I draw a grid based on the data?

**Comment by ogoffart** [comment]
This was just not yet implemented (tracking issue: https://github.com/slint-ui/slint/issues/358 )

As a workaround, you can either nest horizontal and vertical layout, or do the layouting manually by specifying the `x` and `y` properties (like we do in the memory tutorial)
